man/pt.man

```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
+vrms-rpm - relatório de software não grátis instalado
+.SH SYNOPSIS
+\fBvrms-rpm\fR \fI[OPTION]\fR ...
```

Comment:
I see in the manpage that Portuguese has a different word for "option" - it should be changed here, too.